avm/res/cache/redis/linked-servers/main.bicep

```
@@ -0,0 +1,39 @@
+metadata name = 'Redis Cache Linked Servers'
+metadata description = 'This module connects a primary and secondary Redis Cache together for geo-replication.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Primary Redis cache name.')
+param primaryRedisCacheName string
+
+@description('Required. The name of the secondary Redis cache. If not provided, the primary Redis cache name is used.')
+param secondaryRedisCacheName string
+
+@description('Required. The resource ID of the linked server. If not provided, the resource ID of the primary Redis cache is used.')
+param linkedRedisCacheId string
```

As per spec [Inputs - Parameters/Variables for Resource IDs](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr22---category-inputs---parametersvariables-for-resource-ids)

```suggestion
param linkedRedisCacheResourceId string
```

---

Fixed.